core/io/resource_loader.cpp

```diff
@@ -620,6 +620,15 @@ Ref<Resource> ResourceLoader::_load_complete_inner(LoadToken &p_load_token, Erro
 				// Load is in progress, but it's precisely this thread the one in charge.
 				// That means this is a cyclic load.
 				if (r_error) {
+					String error_text = vformat(
+							"Cycle detected during asset loading phase while loading of %s. Workarounds: "
```

Comment:
Use "resource" term for consistency with the rest of the engine:

```suggestion
							"Cycle detected during resource loading phase while loading of %s. Workarounds: "
```